### ISSUE TITLE
Document/decide on approach for appointing new maintainers

### ISSUE THREAD

**Issue by SimonBiggs** [pymedphys/pymedphys#1459]
Hi @sjswerdloff @Matthew-Jennings,

The following document details Jupyter's approach to appointing their equivalent of new maintainers:

https://github.com/jupyter/governance/blob/master/governance.md#council-membership

Their key process for including new maintainers is the following:

> To become eligible for being a [maintainer] an individual must be a Project Contributor who has produced contributions that are substantial in quality and quantity, and sustained over at least one year. Potential [maintainers] are nominated by existing [maintainers] and voted upon by the existing [maintainers]. Based on a successful vote, the newly elected [maintainer] is invited to serve in that capacity.

Is this an approach we would be happy to adopt?

**Comment by sjswerdloff** [comment]
Sure.
There might need to be a maintaining one's maintainer status requirement as well, so you know when to give me the boot with no hard feelings.

**Comment by SimonBiggs** [comment]
> There might need to be a maintaining one's maintainer status requirement as well

Good call. That document also has a clause for that, which seems quite fair to me:

> If a [maintainer] becomes inactive in the project for a period of one year, they will be considered for removal from [maintainership]. Before removal, [the] inactive [maintainer] will be approached by the BDFL to see if they plan on returning to active participation. If not they will be removed immediately upon a [vote by the maintainers]. If they plan on returning to active participation soon, they will be given a grace period of one year. If they don’t return to active participation within that time period they will be removed by vote of the [maintainers] without further grace period. All former [maintainers] can be considered for membership again at any time in the future, like any other Project Contributor. Retired [maintainers] will be listed on the project website, acknowledging the period during which they were active [as maintainers].

**Comment by Matthew-Jennings** [comment]
All sounds good to me!
